### PR TITLE
Add Npred formula to dataset docs

### PR DESCRIPTION
**Pull request by adonath**
**Description**

I noticed we don't show the standard formula of our npred data model. So this pull request add the formula how the predicted counts are computed to the datasets docs page. Another good place would the analysis overview page, but for now it does not matter this much. We can move around the content later...

### CODE REVIEW

File: docs/datasets/index.rst [hunk 1]
```diff
@@ -52,10 +52,28 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
...
+Predicted counts
+================
+Predicted counts are computed like:
+
+.. math::
+
+	N_{Pred} = N_{Bkg} + \mathrm{PSF} \circledast \mathrm{EDISP(\mathcal{E} \cdot F_{Src}(l, b, E_{True}))}
+
+
+Where :math:`$N_{Bkg}$` is the expected residual hadronic background,
```
Comment (registerrier) [inline docs/datasets/index.rst:68]: isn't the bkg also predicted? It is usually a model. We have a npred_background.

File: docs/datasets/index.rst [hunk 2]
```diff
@@ -52,10 +52,28 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
...
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
-
 To analyse multiple runs, you can either stack the datasets together, or perform
 a joint fit across multiple datasets.
 
+
+Predicted counts
+================
+Predicted counts are computed like:
```
Comment (registerrier) [inline docs/datasets/index.rst:61]: For each bin in measured position and energy?

File: docs/datasets/index.rst [hunk 3]
```diff
@@ -52,10 +52,28 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
...
+
+.. math::
+
+	N_{Pred} = N_{Bkg} + \mathrm{PSF} \circledast \mathrm{EDISP(\mathcal{E} \cdot F_{Src}(l, b, E_{True}))}
+
+
+Where :math:`$N_{Bkg}$` is the expected residual hadronic background,
+:math:`$F_{Src}$` the integrated flux of the source model,
```
Comment (registerrier) [inline docs/datasets/index.rst:69]: Normally it should be differential no?

**Review by registerrier — APPROVED** [review]
Thansk @adonath !

You might add the integral (or the sum if you want to represent the discrete integral) over `Etrue`

**Comment by AtreyeeS** [comment]
Thanks @adonath ! Calculation of `npred` is partly explained in IRF theory. https://docs.gammapy.org/dev/irf/theory.html#irf-theory
Do you also want to describe it here?

File: docs/datasets/index.rst (hunk 3 above)
Comment (adonath) [reply docs/datasets/index.rst]: Hm, the spectral model is integrated in energy and the spatial model at least integrated zero order by multiplying with the solid angle...I think one can keep the statement like this.

**Comment by adonath** [comment]
Thanks @registerrier  and @AtreyeeS. I decided to split the formula and make the sum over the source components explicit. I think having this high level description of the predicted counts is still useful, however I also added a link to "IRF theory", where the detailed expressions and integrals are given.

**Comment by AtreyeeS** [comment]
This looks nice now. Do you also want to explicitly mention that the `N_bkg` is not obtained by convolution with the IRFs?

**Comment by adonath** [comment]
I'll proceed and merge this. I'm happy to follow up on it in other PRs...

**Pull request merged**